change/@microsoft-teams-js-f6526430-887f-46bc-bfb4-8c9750214364.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I'd actually mark this change as type `none` instead of patch I think. Should have no noticeable effect on library consumers, only matters if you are building the library yourself.

Reply:
Sure, I agree. Let me correct the change file